.eslintrc.json

```
@@ -0,0 +1,22 @@
+{
+  "env": {
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "plugin:prettier/recommended"
+  ],
+  "ignorePatterns": [".eslintrc.js", "jest.config.js", "/coverage", "/dist"],
```

nit: leftover Jest config file

```suggestion
  "ignorePatterns": [".eslintrc.js", "vitest.config.ts", "/coverage", "/dist"],
```

`.eslintrc.js` also needs to be replaced with either `.eslintrc.json` or `.eslintrc.cjs`.